packages/server-core/src/assets/asset/asset.hooks.ts

```diff
@@ -193,65 +177,74 @@ export const createSceneInStorageProvider = async (context: HookContext<AssetSer
 
   const data = context.data
 
-  // ignore if we are seeding data
-  if (data.assetURL) return
-
-  const storageProvider = getStorageProvider()
-
-  await Promise.all(
-    GLTF_ASSET_FILES.map((ext) =>
-      storageProvider.moveObject(
-        `default${ext}`,
-        `${data.name}${ext}`,
-        `projects/default-project`,
-        context.directory,
-        true
-      )
-    )
-  )
-  try {
-    if (config.server.edgeCachingEnabled)
-      await context.app.service(invalidationPath).create(
-        GLTF_ASSET_FILES.map((file) => {
-          return { path: `${context.directory}${data.name}${file}` }
-        })
-      )
-  } catch (e) {
-    logger.error(e)
-    logger.info(GLTF_ASSET_FILES)
+  // when seeding, no sourceURL is given, and the asset file exists, we just want to add the table entry
+  if (!data.sourceURL) {
+    const fileExists = await context.app.service(fileBrowserPath).get(data.assetURL!)
```

Comment:
When QAing `save as`, file already gets uploaded in `saveSceneGLTF` and `fileExists` returns true. The hook is exited, because of this manifest.json does not update